release_notes/osd_3_3_release_notes.adoc

```diff
@@ -0,0 +1,62 @@
+[[release-notes-osd-3-2-release-notes]]
+= OpenShift Dedicated 3.2 Release Notes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+https://www.openshift.com/dedicated/[OpenShift Dedicated] is a new offering
+based on OpenShift Enterprise that provides a hosted OpenShift 3 environment to
+run the containers powering your applications.
+
+With OpenShift Dedicated, your cluster is isolated from other customers and
+managed by the OpenShift Operations Team. This increases security and
+performance stability.
+
+== Key Features
+
+OpenShift Dedicated offers a dedicated, customized, and managed application
+platform in the public cloud, which is powered by OpenShift 3 and backed by Red
+Hat Operations and Support. Key features include:
```

Comment:
The above is pretty close to just repeating what in the Overview above. Can this be cut down?